### PR TITLE
feat: new readme for Typescript SDK

### PR DESCRIPTION
**Pull request by Arindam200**
## New Readme Preview:

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,375 +1,167 @@
...
+            <picture>
+                <source srcset="./assets/Logo-light-theme.png" media="(prefers-color-scheme: light)">
+                <source srcset="./assets/Logo-dark-theme.png" media="(prefers-color-scheme: dark)">
+                <img src="./assets/Logo-dark-theme.png" height="125" width="600" />
+            </picture>
+        </a><br>
+    </b>
+</h1>
+
+# <p align="center"> Pieces OS Client SDK For TypeScript
```
Comment (shivay-at-pieces) [inline README.md:13]: We can add a badge for the documentation

**Review by shivay-at-pieces — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,375 +1,167 @@
...
 import * as pieces from '@pieces.app/pieces-os-client'
 ```
 
 > **Recommendation**
-> The order that npm packages are saved and added to your dependencies is important and will affect your installation flow. **This slowed me down for quite a bit.**
+> The order that npm packages are saved and added to your dependencies is important and will affect your installation flow. 
 >
 > **If you are having issues with your installation, it is likely due to a conflict in Typescript versions - `npm uninstall typescript` - then go back and perform all other npm installations before reinstalling typescript again**.
 
 You can get it here: [GitHub Repo](https://github.com/pieces-app/example-typescript)
```
[the remark this answers is not in the document]
Comment (Arindam200) [reply README.md]: Yes. Fixed that

**Comment by mason-at-pieces** [comment]
@shivay-at-pieces @Arindam200 let's get this updated like we did the python readme as well. Anything else that needs fixing here?

**Review by shivay-at-pieces — APPROVED** [review]
@mason-at-pieces this looks good to me

**Pull request merged**